app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = current_user.payments
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+    @unpaid_ticket_purchases = current_user.ticket_purchases.unpaid.by_conference(@conference)
+  end
+
+  def create
+    @payment = Payment.new payment_params.merge(stripe_customer_email: params[:stripeEmail],
+                                                stripe_customer_token: params[:stripeToken],
+                                                user: current_user,
+                                                conference: @conference)
```

Comment:
I think if you move the `merge` call to the `payment_params` method, then this line will be cleaner :)
(It will be just `Payment.new payment_params`)